src/SFA.DAS.AssessorService.Web/Controllers/OrganisationSearchController.cs

```diff
@@ -250,43 +252,47 @@ public async Task<IActionResult> DealingWithRequest(OrganisationSearchViewModel
                 {
-                    var isActivelyTrading = await _organisationsApplyApiClient.IsCompanyActivelyTrading(organisationSearchResult.CompanyNumber);
+                    var isActivelyTrading = await _organisationsApiClient.IsCompanyActivelyTrading(organisationSearchResult.CompanyNumber);
 
                     if (!isActivelyTrading)
                     {
                         return View("~/Views/OrganisationSearch/CompanyNotActive.cshtml", viewModel);
                     }
                 }
 
-                var request = CreateAnApplyOrganisationRequest(organisationSearchResult, user);
                 if (organisationSearchResult.OrganisationReferenceType == "RoEPAO")
                 {
-                    //Update assessor organisation status, sync assessor org with apply org and notify org users
-                    await UpdateOrganisationStatus(organisationSearchResult, user);
-                    await TryToCreateOrganisationInApply(request);
-                    await NotifyOrganisationUsers(organisationSearchResult, user);
+                    if (organisationSearchResult.OrganisationIsLive)
+                        //Update assessor organisation status
+                        await UpdateOrganisationStatusAndInvite(organisationSearchResult, user);
                 }
                 else
                 {
-                    //Try creating a contact and an organisation in apply
-                    await _contactsApiClient.MigrateSingleContactToApply(Guid.Parse(signinId));
-                    await _contactsApiClient.UpdateStatus(new UpdateContactStatusRequest(user.Id, ContactStatus.Applying));
-                    await _organisationsApplyApiClient.ConfirmSearchedOrganisation(request);
-                    return Redirect($"{_config.ApplyBaseAddress}/Applications");
-                }
-            }
+                    var request = CreateEpaOrganisationRequest(organisationSearchResult);
+                    request.OrganisationTypeId = viewModel.OrganisationTypeId;
+                    request.Status = OrganisationStatus.Applying;
 
+                    var epaoId = await _organisationsApiClient.CreateEpaOrganisation(request);
+                    _logger.LogInformation($"Organisation with Organisation Id {epaoId.Details} created.");
 
+                    var newOrg = await _organisationsApiClient.GetEpaOrganisation(epaoId.Details);
+                    var response = await _contactsApiClient.UpdateOrgAndStatus(new UpdateContactWithOrgAndStausRequest(user.Id.ToString(),
+                        newOrg.Id.ToString(), null, ContactStatus.Live));
+                    _logger.LogInformation($"Contact with display name {user.DisplayName} is associated with organisation {epaoId.Details}.");
+
+                    return View($"~/Views/Apply/Application/Applications.cshtml");
+                }
+            }
             return View(viewModel);
```

Comment:
Again, quite a lot being handled by this controller.  Push it off into .Application?